recipes/catkin/meta.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: ros-catkin
+  version: 0.7.17
+
+source:
+  - url: https://github.com/ros-gbp/catkin-release/archive/release/melodic/catkin/0.7.17-0.tar.gz
+    fn: ros-catkin-0.7.17.tar.gz
```

Comment:
This line isn't needed
```suggestion
```

Reply:
this is necessary if building more than one package in the same workspace otherwise it can lead to filename clashes in the cache if two packages have the same version number. (note again this is auto-generated so it doesn't really matter that much with regards to work)

Reply:
As in it clashes in the conda-build cache? If so, that's unfortunate.

Reply:
yeah, for example if you check the download name of this package: https://github.com/ros-gbp/genpy-release/releases (first one) it's just `0.6.8.tar.gz`

If you don't have sha256 activated, the next package with `0.6.8.tar.gz` will just assume that it can re-use the cache.

If you have sha256 activated, it will invalidate the cache and re-download.

Both very undesirable situations in my opinion :)

I have this script where I build ~80 ROS packages in parallel and I was running into this bug which is why I added this. 

One way to fix it upstream would be for the source cache to use a more explicit file name or for github to use other file names as just `version.tar.gz` (even though I guess github just uses whatever is the tag which kinda makes sense).